### PR TITLE
Security Fixes

### PR DESCRIPTION
**Pull request by Stratus3D**
[Card](https://rentgroup.atlassian.net/browse/SRV-5995)

* Remove test:examples Rake task as sample apps have been removed
* Remove reference to old Rake task in script/test
* Remove rails and rails-api as dependencies

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,8 +5,6 @@ gemspec
 group :development, :test do
   gem 'pry-byebug'
   gem 'rack-test'
-  gem 'rails'
-  gem 'rails-api'
```
Comment (Stratus3D) [inline Gemfile:9]: I didn't see these referenced anywhere so I removed them. It appears removing these two gems takes care of most of the reported vulnerabilities (they were never real vulnerabilities that could be exploited outside of the dev env).

File: Rakefile [hunk 2]
```diff
@@ -15,12 +15,4 @@ RSpec::Core::RakeTask.new(:spec) do |t|
   t.pattern = 'spec/ops/**/*_spec.rb'
 end
 
-desc 'Executes the example tests'
-task 'test:examples' do
-  %w(rails_app_sample sinatra).each do |ex|
```
Comment (Stratus3D) [inline Rakefile:20]: These sample apps had already been removed so this Rake task didn't work anymore.

File: Rakefile [hunk 3]
```diff
@@ -3,8 +3,8 @@ require 'bundler'
 begin
   Bundler.setup(:default, :development)
 rescue Bundler::BundlerError => e
-  $stderr.puts e.message
-  $stderr.puts 'Run `bundle install` to install missing gems'
+  $stderr.warn e.message
+  $stderr.warn 'Run `bundle install` to install missing gems'
```
Comment (Stratus3D) [inline Rakefile:7]: Rubocop yelled at me about these.

File: script/test [hunk 4]
```diff
@@ -1,19 +1,21 @@
...
 fi
+
 bundle exec rubocop -D $auto_correct
 
-if [[ -n "$1" && $1 != "-a" ]]; then
+if [ -n "$1" ] && [ "$1" = "-a" ]; then
   # pass arguments to test call. This is useful for calling a single test.
   bundle exec rspec "$1"
 else
-  bundle exec rake test:examples && bundle exec rspec
```
Comment (Stratus3D) [inline script/test:18]: `rake test:examples` is gone.

File: script/test [hunk 5]
```diff
@@ -1,19 +1,21 @@
...
 
 echo "===> Running rubocop linter"
-if [[ -n "$1" && $1 == "-a" ]]; then
+if [ -n "$1" ] && [ "$1" = "-a" ]; then
   auto_correct="--auto-correct"
 fi
+
 bundle exec rubocop -D $auto_correct
 
-if [[ -n "$1" && $1 != "-a" ]]; then
```
Comment (Stratus3D) [inline script/test:14]: Shellcheck complained about these.

**Review by pasha — APPROVED** [review]
you might want to bump a [version](https://github.com/rentpath/ops/blob/master/lib/ops/version.rb) at the same time

**Review by joshdayan — APPROVED** [review]

**Review by rkledesma — APPROVED** [review]

**Pull request merged**